services/sidecar/src/sidecar/core.py

```diff
@@ -325,7 +325,13 @@ def process(self):
                                                             environment=self._docker.env,
                                                             nano_cpus=config.SERVICES_MAX_NANO_CPUS,
                                                             mem_limit=config.SERVICES_MAX_MEMORY_BYTES,
-                                                            labels={'user_id': str(self._user_id), 'study_id': str(self._task.project_id), 'node_id': str(self._task.node_id)})
+                                                            labels={
+                                                                'user_id': str(self._user_id),
+                                                                'study_id': str(self._task.project_id),
+                                                                'node_id': str(self._task.node_id),
+                                                                'nano_cpus': str(config.SERVICES_MAX_NANO_CPUS),
+                                                                'mem_limit': str(config.SERVICES_MAX_MEMORY_BYTES)
```

Comment:
could we name the label with something like ```lmit_``` in front? just for clarity.

Reply:
done